ui/components/multichain/token-list-item/token-list-item.js

```diff
@@ -208,4 +269,8 @@ TokenListItem.propTypes = {
    * isOriginalTokenSymbol represents a boolean value to check if the token symbol is original or not
    */
   isOriginalTokenSymbol: PropTypes.bool,
+  /**
+   * isOriginalTokenSymbol represents if this item is the native currency
+   */
+  isNativeCurrency: PropTypes.bool,
```

Comment:
Unfortunately this new prop is required, otherwise the warning symbol shows very briefly *for all tokens* during load.

Reply:
the property name on the comment above the props is wrong  :)